docker-multinode/common.sh

```diff
@@ -474,8 +324,63 @@ kube::multinode::turndown(){
         ;;
     esac
   fi
+
+  # Remove cni0 bridge
+  if [[ ! -z $(ifconfig | grep "cni0") ]]; then
+    ifconfig cni0 down
+    brctl delbr cni0
+  fi
+}
+
+# Detect the OS distro, we support ubuntu, debian, mint, centos, fedora and systemd dist
+kube::multinode::detect_lsb() {
+
+  if kube::helpers::command_exists lsb_release; then
+    lsb_dist="$(lsb_release -si)"
+  elif [[ -r /etc/lsb-release ]]; then
+    lsb_dist="$(. /etc/lsb-release && echo "$DISTRIB_ID")"
+  elif [[ -r /etc/debian_version ]]; then
+    lsb_dist='debian'
+  elif [[ -r /etc/fedora-release ]]; then
+    lsb_dist='fedora'
+  elif [[ -r /etc/os-release ]]; then
+    lsb_dist="$(. /etc/os-release && echo "$ID")"
+  elif kube::helpers::command_exists systemctl; then
+    lsb_dist='systemd'
+  fi
+
+  lsb_dist="$(echo ${lsb_dist} | tr '[:upper:]' '[:lower:]')"
+
+  case "${lsb_dist}" in
+      amzn|centos|debian|ubuntu|systemd)
+        ;;
+      *)
+        kube::log::error "Error: We currently only support ubuntu|debian|amzn|centos|systemd."
+        exit 1
+        ;;
+  esac
+
+  kube::log::status "Detected OS: ${lsb_dist}"
 }
 
+# Install network utils: ifconfig, brctl
+kube::multinode::install_network_utils() {
+    case "${lsb_dist}" in
```

Comment:
Please just check for `yum` or `apt-get` and install the packages if one of those package managers exist.
I'd like to get rid of the OS-specific code